Tensile/KernelWriterAssembly.py

```diff
@@ -1555,12 +1561,19 @@ def initKernel(self, kernel, tPA, tPB ):
     # num vgprs: global -> local elements
     self.numVgprG2LA = 0
     if not kernel["DirectToLdsA"] or self.do["KeepDirectToLdsAlloc"]:
-      self.numVgprG2LA = roundUp((kernel["NumLoadsCoalescedA"] * kernel["NumLoadsPerpendicularA"] *\
-        kernel["GlobalLoadVectorWidthA"] * tPA["bpe"])/(float)(self.bpr))
+      self.numVgprG2LA = roundUp((kernel["NumLoadsCoalescedA"] * kernel["NumLoadsPerpendicularA"] * \
+        kernel["GlobalLoadVectorWidthA"] * tPA["bpe"]) / (float)(self.bpr))
+    # using ds_write_b8: need one more vgpr space to do lshr
+    if self.localWriteInstructionA.blockWidth == 0.25:
+      self.numVgprG2LA = self.numVgprG2LA * 2
+
```

Comment:
we use 

> Yes, this would be the better solution to avoid the mess for checkin/out in i8-local-write ... but have you observed any noticeable perf drop caused by the increasing of vgpr ? Hopefully not.

We use less vgpr than check-in/out due to check-in/out method do multiple/redundant checkout.